src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -65,14 +65,17 @@ parseTypeSynonymDeclaration =
 
 parseValueDeclaration :: TokenParser Declaration
 parseValueDeclaration = do
-  name <- parseIdent
-  binders <- P.many parseBinderNoParens
-  value <- Left <$> (indented *>
-                       P.many1 ((,) <$> parseGuard
-                                    <*> (indented *> equals *> parseValueWithWhereClause)
-                               ))
-       <|> Right <$> (indented *> equals *> parseValueWithWhereClause)
-  return $ ValueDeclaration name Public binders value
+  binders <- (:) <$> parseBinderAtom <*> (P.many parseBinderNoParens)
```

Comment:
When using `parseBinder` instead of `paresBinder`, modules in the test directory seem like failing to build. I copied the error in the following gist:

https://gist.github.com/noraesae/f9b2b0e1dfc8514d468b09a14bc149a7

Do you have any idea?